# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1450919765 18000
#      Wed Dec 23 20:16:05 2015 -0500
# Node ID 22318eeaca3b44482a7756344f5e004046482f3f
# Parent  34d0497730e6f133ef430e782e354b3fb19bd0f1
Add gnuplot "canvas" terminal to the print.m doc-string.
(Part of Bug #44187).

* print.m: Include the gnuplot "canvas" device in the doc-string. Use
consistent phrasing "only available for the Gnuplot graphics toolkit".

diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -138,16 +138,20 @@
 ## file, @file{@var{filename}.(eps|pdf)}, for the graph portion of the plot.
 ## The @samp{standalone} variants behave as described for
 ## @samp{epslatexstandalone} above.
 ##
 ##   @item  ill
 ##   @itemx aifm
 ##     Adobe Illustrator (Obsolete for Gnuplot versions > 4.2)
 ##
+##   @item canvas
+##     Javascript-based drawing on HTML5 canvas viewable in a web browser
+## (only available for the Gnuplot graphics toolkit).
+##
 ##   @item  cdr
 ##   @itemx @nospell{corel}
 ##     CorelDraw
 ##
 ##   @item dxf
 ##     AutoCAD
 ##
 ##   @item  emf
@@ -188,17 +192,17 @@
 ##   @item svg
 ##     Scalable vector graphics
 ##
 ##   @item tikz
 ##   @itemx tikzstandalone
 ##     Generate a @LaTeX{} file using PGF/TikZ@.  For the FLTK toolkit
 ## the result is PGF. The @samp{tikzstandalone} device produces a @LaTeX{}
 ## document which includes the TikZ file (@samp{tikzstandalone} is only
-## available with the gnuplot graphics toolkit).
+## available for the Gnuplot graphics toolkit).
 ##
 ##   If the device is omitted, it is inferred from the file extension,
 ## or if there is no filename it is sent to the printer as PostScript.
 ##
 ## @item -d@var{ghostscript_device}
 ##   Additional devices are supported by Ghostscript.
 ## Some examples are;
 ##
